charts/consul/templates/gateway-resources-job.yaml

```diff
@@ -1,3 +1,4 @@
+{{ template "consul.validate_deprecated_apigateway_not_set" . }}
```

Comment:
I think this is the best place to put this?

Reply:
I'd say here or in connect-inject-deployment where we moved the stanza to.